cmake/test-targets.cmake

```diff
@@ -156,7 +162,7 @@ if (COVER_PATH AND PROVE_PATH)
         WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}"
     )
     add_dependencies(coverage-codecovbash symlinks)
-
+    add_custom_target(coverage-ci)
```

Comment:
This is still a draft but adding a custom target without dependencies won't do much.